### PR TITLE
Feature flutter

### PR DESCRIPTION
**Pull request by Piyush7034**
[RCF-167] [RCF-250] Auditing and Logging Events

[RCF-167]: https://mosip.atlassian.net/browse/RCF-167?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[RCF-250]: https://mosip.atlassian.net/browse/RCF-250?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: android/app/src/main/java/io/mosip/registration_client/api_services/BiometricsDetailsApi.java [hunk 1]
```diff
@@ -390,23 +378,31 @@ public void conditionalBioAttributeValidation(@NonNull String fieldId, @NonNull
                List<BiometricsDto> temp= registrationDto.getBestBiometrics(fieldId,modality);
                biometricsDtoList.addAll(temp);
             }
+            System.out.println(biometricsDtoList);
```
Comment (jainhitesh9998) [inline android/app/src/main/java/io/mosip/registration_client/api_services/BiometricsDetailsApi.java:381]: remove system.out.println

File: lib/platform_android/network_service_impl.dart [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+import 'package:registration_client/platform_spi/network_service.dart';
+import 'package:http/http.dart' as http;
+
+class NetworkServiceImpl implements NetworkService {
+  @override
+  Future<String> checkInternetConnection() async {
+
+    try {
+      final response = await http.get(Uri.parse(
+          'https://api-internal.dev2.mosip.net/v1/syncdata/actuator/health'))
```
Comment (jainhitesh9998) [inline lib/platform_android/network_service_impl.dart:11]: make this configurable from parameter, while qa-triad env is called it'll refer to dev2

File: lib/provider/connectivity_provider.dart [hunk 3]
```diff
@@ -1,26 +1,53 @@
...
+  bool _isConnected = false;
 
   ConnectivityResult get connectivityResult => _connectivityResult;
+  bool get isConnected => _isConnected;
 
-  ConnectivityProvider() {
-    initialize();
-  }
+  // ConnectivityProvider() {
+  //   initialize();
```
Comment (jainhitesh9998) [inline lib/provider/connectivity_provider.dart:14]: remove commented lines

File: lib/provider/connectivity_provider.dart [hunk 4]
```diff
@@ -1,26 +1,53 @@
...
 
-  Future<void> initialize() async {
-    final connectivity = Connectivity();
-    _connectivityResult = await connectivity.checkConnectivity();
+  // Future<void> initialize() async {
+  //   late ConnectivityResult result;
+  //   try {
+  //     result = await _connectivity.checkConnectivity();
+  //   } on PlatformException catch (e) {
+  //     debugPrint('Couldn\'t check connectivity status $e');
```
Comment (jainhitesh9998) [inline lib/provider/connectivity_provider.dart:23]: remove commented lines

File: lib/provider/connectivity_provider.dart [hunk 5]
```diff
@@ -1,26 +1,53 @@
...
+  // }
 
-  bool get isConnected => _connectivityResult != ConnectivityResult.none;
+  // void updateConnection(ConnectivityResult result) async {
+  //   _connectivityResult = result;
+  //   String response = await networkService.checkInternetConnection();
+  //   if(response == "200") {
+  //     log("Back online");
+  //     _vpnConnection = true;
+  //   } else {
```
Comment (jainhitesh9998) [inline lib/provider/connectivity_provider.dart:36]: remove commented lines

File: lib/ui/process_ui/widgets/date_control.dart [hunk 6]
```diff
@@ -0,0 +1,121 @@
...
+    super.initState();
+  }
+
+  void _selectDate() async {
+    pickedDate = await showDatePicker(
+        context: context,
+        initialDate: DateTime.now(),
+        firstDate: DateTime(
+            2000),
+        lastDate: DateTime(2101));
```
Comment (jainhitesh9998) [inline lib/ui/process_ui/widgets/date_control.dart:38]: any reason we use last date and first date as these specific values

**Review by jainhitesh9998 — CHANGES_REQUESTED** [review]

File: android/app/src/main/java/io/mosip/registration_client/api_services/BiometricsDetailsApi.java (hunk 1 above)
Comment (Piyush7034) [reply android/app/src/main/java/io/mosip/registration_client/api_services/BiometricsDetailsApi.java]: Removed sout.

File: android/ajcore.20230922.161537.245.txt [hunk 7]
```diff
@@ -0,0 +1,554 @@
+---- AspectJ Properties ---
```
[the remark this answers is not in the document]
Comment (Piyush7034) [reply android/ajcore.20230922.161537.245.txt]: Removed this file

File: lib/platform_android/network_service_impl.dart (hunk 2 above)
Comment (Piyush7034) [reply lib/platform_android/network_service_impl.dart]: Using it on flutter side so added it with the constants. For qa-triad api url, it is declared as baseUrl in build.gradle file as it is used on android side.

File: lib/provider/connectivity_provider.dart (hunk 3 above)
Comment (Piyush7034) [reply lib/provider/connectivity_provider.dart]: Removed comments.

File: lib/provider/connectivity_provider.dart (hunk 4 above)
Comment (Piyush7034) [reply lib/provider/connectivity_provider.dart]: Removed comments.

File: lib/provider/connectivity_provider.dart (hunk 5 above)
Comment (Piyush7034) [reply lib/provider/connectivity_provider.dart]: Removed comments.

File: lib/ui/process_ui/widgets/date_control.dart (hunk 6 above)
Comment (Piyush7034) [reply lib/ui/process_ui/widgets/date_control.dart]: Not given proper validation so changed last date with current year and first date with 100 yrs diff from curr date

File: lib/platform_android/network_service_impl.dart (hunk 2 above)
Comment (jainhitesh9998) [reply lib/platform_android/network_service_impl.dart]: presently we are picking it from app_config.dart, we can replace it's location in future

File: lib/ui/process_ui/widgets/date_control.dart (hunk 6 above)
Comment (jainhitesh9998) [reply lib/ui/process_ui/widgets/date_control.dart]: ok let's check what's needed here.

**Comment by jainhitesh9998** [comment]
check what validation is needed for dates.
merging the PR for now.

**Comment by Piyush7034** [comment]
Yes, will check

**Pull request merged**